app/controllers/games_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+# Comment necessary for Rubocop
+class GamesController < ApplicationController
+  before_action :set_game, only: %i[show update destroy]
+
+  # GET /games
+  def index
+    @games = Game.all
+
+    render json: @games
+  end
+
+  # GET /games/1
+  def show
+    render json: render_game_with_state_and_player(@game)
+  end
+
+  # POST /games
+  def create
+    Game.delete_all
+    new_game = TicTacToeLogic::GameBoardData.new
+    @game = Game.new({ "board": new_game })
```

Comment:
It feels weird that you are creating a new_game in the controller and then passing the into `Game.new` 
Is there a way to make it so that Game.new() will create a new board if one isn't supplied? 
```suggestion
    @game = Game.new()
```

Reply:
Im not sure I understand this. The 'Game' model, when created, needs to be passed in a 'board' field since it is in its schema. Like it can't be blank like so: 'Game.new()'

In sinatra, the controller is also where I created the board instance. So it should not be in this file?

Reply:
okay. That makes sense. Can we remove the temp variable at least. 
```suggestion
    @game = Game.new({ "board": TicTacToeLogic::GameBoardData.new })
```

Reply:
👍🏽